projects/cdk/schematics/ng-update/v4/steps/migrate-appearance.ts

```diff
@@ -0,0 +1,31 @@
+/// <reference lib="es2021" />
+import {getActiveProject, saveActiveProject} from 'ng-morph';
+import type {DevkitFileSystem} from 'ng-morph/project/classes/devkit-file-system';
+
+import type {TuiSchema} from '../../../ng-add/schema';
+import {infoLog, REPLACE_SYMBOL, SMALL_TAB_SYMBOL} from '../../../utils/colored-log';
+
+export function migrateAppearance(
```

Comment:
Could we just reuse this utility ?

https://github.com/taiga-family/taiga-ui/blob/178a5d381901d73db5f5a3777a5ccdc89d2d8863/projects/cdk/schematics/ng-update/steps/replace-enums.ts#L16-L20

And do something similar to the following lines:
https://github.com/taiga-family/taiga-ui/blob/178a5d381901d73db5f5a3777a5ccdc89d2d8863/projects/cdk/schematics/ng-update/v3/constants/enums.ts#L3-L14